src/components/AddFoodForm.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { Input, Button } from 'antd';
+
+function AddFoodForm({createFood}) {
+    const [name, setName] = useState('');
+    const [image, setImage] = useState('');
+    const [calories, setCalories] = useState(1);
+    const [servings, setServings] = useState(1);
+
+   
+    const handleName= (e) => setName(e.target.value);    
+    const handleImage = (e) => setImage(e.target.value);
+    const handleCalories = (e) => setCalories(e.target.value);
+    const handleServings = (e) => setServings(e.target.value);
+
+  
+
+    const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const newFood = { name, image, calories, servings };
+    console.log(newFood);
+
+
+    createFood(newFood);
+
+    
+    setName('');
+    setImage('');
+    setCalories(1);
+    setServings(1);
+  };
+
+  return (
+    <div>
+        
+        <h3>Add New Food</h3>
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="name">Name</label>
+        <Input name='name' type="text" onChange={{handleName}}/>
```

Comment:
In every 'Input' you are calling the 'handleFunction' like this:
`<Input name='name' type="text" onChange={{handleName}}/>`
In this way you are wrapping the function in an object, and doing so the 'onChange' will not be able to run the function, this is why every time you create a food it appears without the name neither the image that you passed. To correct this you just need to take out one pair of '{' like this:
`<Input name='name' type="text" onChange={handleName}/>`